x/oracle/keeper/task.go

```diff
@@ -66,24 +66,21 @@ func (k Keeper) SetClosingBlockStore(ctx sdk.Context, task types.TaskI) {
 		store.Set(types.ClosingTaskIDsStoreKey(task.ExpireHeight), bz)
 		return
 	case *types.TxTask:
-		store.Set(types.ClosingTaskIDsTimedStoreKey(task.ValidTime), bz)
+		store.Set(types.TimeStoreKey(types.ClosingTaskStoreKeyTimedPrefix, task.ValidTime), bz)
```

Comment:
Does `genesis` also need this? @kevin-yuhh

Reply:
yes, genesis will init valid tx task and task